### DISCUSSION TITLE
warning: remote HEAD refers to nonexistent ref, unable to checkout

### DISCUSSION THREAD

**Discussion by mariusa** [piku/piku#296] [Q&A]
Hi, getting this when committing 1st time an app to piku. If it's normal, I wonder if it can be suppressed? eg by having piku check if it's a new app which needs to be created and skip the command which gives that warning?

**Comment by chr15m** [comment]
No I don't think that's normal, I have not seen it before on any of my installs. Perhaps has something to do with the way you have set up Piku.
